UnitedIncome/components
# [APP 4720] - Life Insurance selection screen

### Description
> Please provide a brief description of your changes below.

Added two new illustrations for the life insurance selection screen - UmbrellaCircleIllustration and InsuranceCircleIllustration

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

check to make sure they look nice and pretty.

### Screenshots
> If this pull request includes a visual change please attach any applicable screenshots.

### Comments
> Any additional comments that will aid with the review process.

- [JIRA Ticket](https://unitedincome.atlassian.net/browse/app-4720)
